docs/js/guides/bas-external-system.md

```diff
@@ -0,0 +1,165 @@
+
+However, due to the subscription between the BAS and the CF account there is a connection from the local application to the SAP S/4HANA system.
+On a high level the connection works the following way:
+- The BAS includes an HTTP_PROXY running at http://localhost:8887 
+- This proxy forwards all http requests to the CF account
+- CF searches the existing destinations for one matching the URL of the request
+- If a destination is present, an initial request is sent to this destination
+- The proxy is a reverse proxy also piping back the response to the BAS
+
+The SDK helps you to consider the HTTP_PROXY automatically and makes it easy to use the same code base locally and in production on CF.
+As described in [the destination lookup](../features/connectivity/destination-js-sdk) the SDK has a destination lookup priority considering environment variables first.
+
+The trick is to define a `destinations` environment variable when you run locally, which works like a switch under the hood when you execute:
+
+```$xslt
+executeHttpRequest({ destinationName: "myDestinationName", jwt: "myJWT" });
+```
+
+The code is the same for local execution and production. The two situation are:
+
+**Case 1:** When run locally, the SDK reads the destination from the environment variables. The `jwt` is irrelevant.
+The destination contains only the name and URL of the real CF destination. 
+Since the destination has no proxy type specified, the SDK takes the HTTP_PROXY into account, as is the default.
+From there the flow described above takes place.
+
+**Case 2:** When run on CF there is no environment variable present. 
+The `jwt` is used to fetch the full destination from the service.
+The proxy type is `OnPremise` and the connectivity service provides all proxy information.
+
+
+:::note
+The `executeHttpRequest()` function is used by all request builders provided of the SDK as well as by CAP applications connecting to an SAP S/4HANA system.
+:::
+
+## Setup ##
+
+### Prerequisite I: The Business Application Studio ###
+
+You have a Cloud Foundry account and a subscription to the BAS. 
+Start the BAS and connect your BAS workspace to the CF account. 
+This is done via the little CF icon (<img src={useBaseUrl('img/cf-connect-button.jpg')} />) on the left of the BAS.
+The connection enables the proxy connection from the BAS to your CF account.
+
+### Prerequisite II: Cloud Connector and Destination Setup  <a name="CCandDestSetup"></a> ###
+
+You have a working Cloud Connector setup and in your account contains a destination pointing to an On-Premise system you want to connect to.
+Per default, destinations are not usable by a connected BAS. 
+You need to set two properties `WebIDEEnabled` and `HTML5.DynamicDestination` to enable that feature for a specific destination.
+Go to the destination configuration in CF and add the properties:
+
+|Property|Value|
+|---|---|
+|WebIDEEnabled|true|
+|HTML5.DynamicDestination|true|
+  
+### Local Application Setup
```

Comment:
Maybe?
```suggestion
### Local Application Setup with Visual Studio Code
```

Reply:
technically it is not VSC but UX is almost the same. I would call it `Local Application Setup wit Business Application Studio`?

Reply:
Aaaah, I was sleeping obviously. Then I guess this is irrelevant.